pkg/worker/worker.go

```diff
@@ -440,14 +440,6 @@ func (s *Worker) shutdown() error {
 	// Stops goroutines
 	s.cancel()
 
-	// Stop all running containers forcefully
-	s.containerInstances.Range(func(containerId string, _ *ContainerInstance) bool {
-		if err := s.stopContainer(containerId, true); err != nil {
-			log.Printf("Failed to stop container: %v\n", err)
-		}
-		return true // continue iterating
-	})
```

Comment:
We don't need to stop containers here. Let them stop themselves when the global context is cancelled.

Reply:
👍